containers/src/Kokkos_DynamicView.hpp

```diff
@@ -299,19 +299,21 @@ class DynamicView : public Kokkos::ViewTraits<DataType, P...> {
 
     // *m_chunks[m_chunk_max] stores the current number of chunks being used
     uintptr_t* const pc = reinterpret_cast<uintptr_t*>(m_chunks + m_chunk_max);
-
+    std::string _label =
```

Comment:
I am actually curious if GCC 5.3.0 still complains if you drop the underscore prefix.

Reply:
c.f. https://github.com/kokkos/kokkos/pull/3315#discussion_r477609994